### PR TITLE
[FEAT-414] Allow user to add the PR description

### PR DESCRIPTION
**Pull request by JonasTeixeira42**
This PR solves issue #414

- It creates a new page so that the user can add the PR description 
image
image

### CODE REVIEW

File: app/assets/builds/tailwind.css
Comment (adbatista) [inline app/assets/builds/tailwind.css:1]: this file can be removed

File: app/models/contribution.rb [hunk 1]
```diff
@@ -64,4 +66,10 @@ def update_rejected_reason(rejected_reason = "other_reason", reviewer_id)
   scope :valid_pull_requests, -> { where.not(state: :refused) }
   scope :without_pr_state, ->(state) { where.not(pr_state: state) }
   scope :tracking, -> { where(tracking: true) }
+
+  private
+
+  def normalize_description_blank_values
+    self[:description] = description.blank? ? nil : description
```
Comment (adbatista) [inline app/models/contribution.rb:73]: you can do here `self[:description] = description.presence`

File: spec/controllers/contributions_controller_spec.rb [hunk 2]
```diff
@@ -45,9 +47,103 @@
...
+    end
+  end
+
+  describe 'PUT update' do
+    let(:contribution) { create(:contribution) }
+    let(:params) do
+      {
+        id: contribution.id,
+        contribution: {
+          description:
```
Comment (adbatista) [inline spec/controllers/contributions_controller_spec.rb:92]: add the `description: description` just to be consistent with the other parts of the code

File: app/models/contribution.rb (hunk 1 above)
Comment (JonasTeixeira42) [reply app/models/contribution.rb:73]: on it

File: app/views/contributions/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<ul class="breadcrumb">
+  <li class="breadcrumb-item"><%= link_to t(:back, scope: :helpers), contributions_path %></li>
+</ul>
+
+<h1>
+  <%= t(:editing, scope: %i(helpers actions), model: Contribution.model_name.human) %>
```
Comment (adbatista) [inline app/views/contributions/edit.html.erb:6]: here should be something like "Add description" or something like

**Review by adbatista — APPROVED** [review]

**Pull request merged**